browser/src/control/Permission.js

```diff
@@ -68,7 +68,7 @@ L.Map.include({
 				button.on('click', function () {
 					that._requestFileCopy();
 				});
-			} else if (!this.options.canTryLock && (window.mode.isMobile() || window.mode.isTablet())) {
+			} else if (!this['wopi'].UserCanWrite || (!this.options.canTryLock && (window.mode.isMobile() || window.mode.isTablet()))) {
```

Comment:
Suggestion: For check read-only mode, we can use `map.isReadOnlyMode()`

Reply:
Now I question the state of `this.wopi` on Mobile.

Reply:
Updated the test.

Reply:
I'm actually surprised. If we are in readonly mode, then why should we check the condition of UserCanWrite? By reading the code, it seems illogical. Correct me if I'm wrong.

Reply:
you can be in readonly mode because you open a .csv or .txt. See `readonlyStartingFormats`.

Then when you click the edit button, if `this['wopi'].UserCanNotWriteRelative` is false, then you get offered to save as ODF or to edit with possible losses. But the user can't write at all, there is no point.

The original issue is "we open a .cvs and the edit button is there, but it does nothing", whith `UserCanNotWriteRelative` set to true, and `UserCanWrite` to false.